content/en/index.md

```diff
@@ -7,163 +7,169 @@ url: /
+
+<video controls>
+  <source src="/assets/hotb-drone-night.webm" type="video/webm">
+  <source src="/assets/hotb-drone-night.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+<span>Video from #hotb19 🎥 <a href="https://martinmoerke.de/" target="_blank">@martinmoerke</a></span>
 
-TODO: Drohnen Video einfügen?
-    
 ## Programm
     
-### Friday - Day 1
+### Friday - Day 1 - 27.09.2024
 
 | When | What |
 | :-- | :-- |
 | 11:42 | Set-up |
-| 18:42 | Opening |
-| 19:00 | Introduction 3D-Printing |
-| 20:00 | Hot-Dog Buffet TBA |
-| HH:MM | End of Day 1 |
+| 18:42 | Official Opening |
+| TBD | Introduction 3D-Printing |
+| TBD | Food |
+| 02:30 | End of Day 1 (02:30 28.09.2024) |
 
-### Saturday - Day 2
+### Saturday - Day 2 - 28.09.2024
 
 | When | What |
 | :-- | :-- |
-| HH:MM | Doors Opening Day 2 |
-| 11:00 | Breakfast? |
-| 14:00 | Jugendtreff??? |
-| 17:00 | Introduction Soldering |
-| 18:00 | Food? |
-| HH:MM | End of Day 1 |
+| TBD | Doors Opening Day 2 |
+| TBD | Breakfast |
+| TBD | Jugend-Aktion |
+| TBD | Introduction Soldering |
+| TBD | Food |
+| 02:30 | End of Day 2 (02:30 29.09.2024) |
 
-### Sunday - Day 3
+### Sunday - Day 3 - 29.09.2024
 
 | When | What |
 | :-- | :-- |
-| HH:MM | Doors Opening Day 3 |
-| 12:15 | [Closing & Goodbye](https://pretalx.chaostreff-flensburg.de/hoth2023/talk/DND3DV/) |
+| TBD | Doors Opening Day 3 |
+| TBD | Breakfast |
+| 12:15 | Closing & Goodbye |
 | until ready | Dismantling |
-| 16:00 | Übergabe Location - Alles muss aufgeräumt und sauber sein |
+| 16:00 | Handover location - everything must be tidy and clean |
 
-:arrow_right: The programme is always WIP (Work in Progress).
+TBD = In Planung.
     
-Während der gesamten Zeit werden eine 3D-Druckstation und eine Lötstation zur Verfügung stehen.
+A 3D printing station and a soldering station will be available throughout the event.
 
 ## FAQ
 
-Du hast noch mehr Fragen? Schreibe uns gerne eine E-Mail an hoth@chaostreff-flensburg.de
+Do you have any more questions? Please send us an e-mail to hotb@chaostreff-flensburg.de
 
-### An wen richtet sich die Veranstaltung?
-Die Veranstaltung richtet sich an alle, die sich für Basteln, Programmieren, Hacken, Tüfteln, Löten, 3D-Drucken, Chaos und Netzpolitik interessieren oder begeistern können. Du musst kein Experte\*in sein oder Vorkenntnisse haben, um an dieser Veranstaltung teilzunehmen.
-**Am besten schaust du einfach mal vorbei.**
+### Who is the event for?
+The event is aimed at anyone who is interested in or enthusiastic about tinkering, programming, hacking, tinkering, soldering, 3D printing, chaos and net politics. You don't have to be an expert or have any prior knowledge to take part in this event.
+**The best thing to do is just drop by.**
 
-Du solltest nicht kommen, wenn du an Verschwörungsmythen glaubst oder nicht den Code of Conduct einhälst.
+You should not come if you believe in conspiracy myths or do not adhere to the [Code of Conduct](/coc/).
```

Comment:
haben wir einen englischen Code of Conduct? Sollten wir den direkt verlinken? (Sprungmarke?)

Reply:
Nur die Deutsche Sprachversion braucht die language in der URL. Habe ich hinzugefügt